frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FeatureLifecycleTooltip.tsx

```diff
@@ -319,6 +325,7 @@ const SafeToArchive: FC<{
                 sx={{
                     display: 'flex',
                     flexDirection: 'row',
+                    flexWrap: 'wrap',
```

Comment:
when the padlock is added the buttons should not grow in size but go to the next line